http/jsonapi/generator/internal/pay/open-api_test.go

```diff
@@ -110,9 +132,20 @@ func GetPaymentMethodsHandler(service Service) http.Handler {
 CreatePaymentMethodSEPAHandler handles request/response marshaling and validation for
  Post /beta/payment-methods/sepa-direct-debit
 */
-func CreatePaymentMethodSEPAHandler(service Service) http.Handler {
+func CreatePaymentMethodSEPAHandler(service Service, authBackend AuthorizationBackend) http.Handler {
 	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
 		defer errors.HandleRequest("CreatePaymentMethodSEPAHandler", w, r)
+		// Only do Authentication if a authentication Backend is available. Otherwise this is handled somewhere else
+		if authBackend != nil {
```

Comment:
I think `authBackend` can be assumed to never be `nil`, if the `Router()` function is removed as I suggested.